spec/spec_helper.rb

```diff
@@ -16,6 +16,7 @@
   config.before(:each) do
     TEST_REDIS.flushdb
     Sqeduler::Service.config = nil
+    REDIS_CONFIG = REDIS_CONFIG.dup
```

Comment:
Good question! The reason this is here is that it allows the `REDIS_CONFIG` hash above to be frozen and skirts around the problem that `sidekiq` [immediately modifies the redis configuration](https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/redis_connection.rb#L10) hash when you try to set it
```ruby
module Sidekiq
  class RedisConnection
    class << self

      def create(options={})
        options[:url] ||= determine_redis_provider
...
```
when squeduler [tries to set in in service.rb](https://github.com/square/sqeduler/blob/c08b7350227cea38a0eaf62e37dcf68560dbb2bf/lib/sqeduler/service.rb#L67)
```ruby
def setup_sidekiq_redis(config)
  return if Service.config.redis_hash.nil? || Service.config.redis_hash.empty?
  config.redis = Service.config.redis_hash
end
```
I'm going to move the `REDIS_CONFIG` hash values into the `TEST_REDIS` declaration and `dup` the `REDIS_CONFIG` values in this `before` block unless you say otherwise.